<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>120</width>
      <height>511</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>509</height>
      </rect>
     </property>
     <widget class="QPushButton" name="glock_weapon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Glock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ak47_weapon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>AK-47</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m3_weapon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>M3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="awp_weapon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>AWP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="base_dust">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Dust</string>
      </property>
     </widget>
     <widget class="QPushButton" name="base_inferno">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Inferno</string>
      </property>
     </widget>
     <widget class="QPushButton" name="base_aztec">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Aztec</string>
      </property>
     </widget>
     <widget class="QPushButton" name="box_brown">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Box Brown</string>
      </property>
     </widget>
     <widget class="QPushButton" name="box_black">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Box Black</string>
      </property>
     </widget>
     <widget class="QPushButton" name="box_wood">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Box Wood</string>
      </property>
     </widget>
     <widget class="QPushButton" name="box_wood_metal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Box Wood Metal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="box_metal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Box Metal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bomb_area_a">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>360</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Bomb Area A</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bomb_area_c">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Bomb Area C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="T_spawn">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>450</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Terrorist Spawn</string>
      </property>
     </widget>
     <widget class="QPushButton" name="C_spawn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>480</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Counter Spawn</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bomb_area_b">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>390</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Bomb Area B</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="map">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
